module/app/app.go

```diff
@@ -705,7 +708,7 @@ func NewGravityApp(
 		BankKeeper:      bankKeeper,
 		FeegrantKeeper:  nil,
 		SignModeHandler: encodingConfig.TxConfig.SignModeHandler(),
-		SigGasConsumer:  ante.DefaultSigVerificationGasConsumer,
+		SigGasConsumer:  ethante.DefaultSigVerificationGasConsumer,
```

Comment:
Flagging this because it looks off to my naive eyes, shouldn't we have a new third handler of our own and we dispatch to the correct one there?